### PR TITLE
CompRobotDrives

### PR DESCRIPTION
**Pull request by WhyDoesGodDoThis**
Now works with compotition robot driving

### CODE REVIEW

File: src/main/java/frc/robot/RobotContext.java [hunk 1]
```diff
@@ -98,12 +97,7 @@ public void periodic() {
         
         if ( robotState.getScoringLocation().isPresent()) {
         	ScoringLocation scoreloc = robotState.getScoringLocation().get();
-        	Pose2d absoluteScoringPose = scoreloc.computeAbsolutePose();
-        	
-        	Pose2d realRobotPose = estimatedRobotPose.get();
```
Comment (dcowden) [inline src/main/java/frc/robot/RobotContext.java:103]: why are we removing this code?

**Comment by dcowden** [comment]
I dont think we can merge these changes to RobotContext-- its removing code we need

**Review by mandrews281 — COMMENTED** [review]
I think the only files that should be updated are the DriveSubsystem and the RobotConstants.  Everything else isn't related to getting the competition robot running.

**Comment by dcowden** [comment]
ok will grab the bits we need tomorrow.

**Comment by dcowden** [comment]
i think is/was behind main: we cherrypicked the main commit onto https://github.com/entech281/Robot2023/pull/65

**Pull request closed without merging**